client/components/search/index.jsx

```diff
@@ -38,7 +38,8 @@ module.exports = React.createClass( {
 		onKeyDown: React.PropTypes.func,
 		disableAutocorrect: React.PropTypes.bool,
 		onBlur: React.PropTypes.func,
-		searching: React.PropTypes.bool
+		searching: React.PropTypes.bool,
+		forceCloseButton: React.PropTypes.bool
```

Comment:
The x is the only way to close the search — which hides the other functions on the page like adding a new site. Without the x (or the `forceCloseButton` prop) there's no way to exit "search mode."